### DISCUSSION TITLE
User requirements repository

### DISCUSSION THREAD

**Discussion by BerkvensNick** [soilwise-he/SoilWise-documentation#24] [Ideas]
regarding the elicitation/identification of new user requirements, where is the best place to start collecting them?

- in a new repository?
- or in the repository Soilwise-userstories (https://github.com/soilwise-he/Soilwise-userstories)?

@pvgenuchten @chlebas @TunaCoppens

**Comment by BerkvensNick** [comment]
Somakanthan would chose a seperate/dedicated repository, because a lot of work has already been done in repository Soilwise-userstories

but we have to be sure the link with the use cases and validation is kept

**Comment by pvgenuchten** [comment]
I would leave that to the user case lead, some may prefer a github repository, others excel on sharpoint.
